### PR TITLE
feat(normalize_storage): use `MINIO_BUCKET` env-var

### PR DESCRIPTION
**Pull request by drgarcia1986**
Use MINIO_BUCKET environment variable to define the bucket name for minio storage type
instead of hardcoded `git`

If there isnt a value on MINIO_BUCKET var, normalize_storage will use `git` by default

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

**Comment by drgarcia1986** [comment]
related https://github.com/deis/slugrunner/pull/66

**Comment by mboersma** [comment]
Jenkins, add to whitelist.

**Comment by mboersma** [comment]
Looks like `shellcheck` complained about some

**Comment by mboersma** [comment]
Looks like `shellcheck` complained about not quoting a variable. Sorry--it's pretty strict but obviously an easy fix.

```
09:30:32 
09:30:32 In rootfs/bin/normalize_storage line 6:
09:30:32   echo $MINION_BUCKET_NAME > /app/objectstore/minio/builder-bucket
09:30:32        ^-- SC2086: Double quote to prevent globbing and word splitting.
09:30:32 
09:30:32 Makefile:56: recipe for target 'test-style' failed
```

**Comment by drgarcia1986** [comment]
Oh sure, make sense, sorry my bad!
Fixed!

File: rootfs/bin/normalize_storage [hunk 1]
```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 export BUCKET_FILE=/var/run/secrets/deis/objectstore/creds/builder-bucket
 if [ "$BUILDER_STORAGE" == "minio" ]; then
+  MINION_BUCKET_NAME=${MINIO_BUCKET:-git}
```
Comment (mboersma) [inline rootfs/bin/normalize_storage]: Could you make this `MINIO_BUCKET_NAME` (not `MINION`)?

Comment (drgarcia1986) [reply rootfs/bin/normalize_storage]: Done

**Comment by drgarcia1986** [comment]
Hi @mboersma , Is there something then I need to fix?

**Comment by mboersma** [comment]
@drgarcia1986 not your problem as far as I can tell, it looks like we have some issues with the e2e tests. I'll be looking at that today and get it sorted out.

**Comment by drgarcia1986** [comment]
@mboersma tks 👍

**Comment by mboersma** [comment]
Jenkins, test this please.

**Comment by drgarcia1986** [comment]
c'mon jenkins, be nice to me :/

**Review by mboersma — APPROVED** [review]

**Review by vdice — APPROVED** [review]

**Pull request merged**